contentcuration/contentcuration/static/js/edit_channel/router.js

```diff
@@ -39,15 +40,19 @@ ChannelEditRouter  = Backbone.Router.extend({
   },
 
 	edit_page : function(){
-		this.open_channel(true, false, window.current_channel.get_root("main_tree"));
+		this.open_channel(true, false, false, window.current_channel.get_root("main_tree"));
+	},
+	staging_page: function(){
+		this.open_channel(true, false, true, window.current_channel.get_root("staging_tree"));
 	},
 	preview_page : function(){
-		this.open_channel(false, false, window.current_channel.get_root("main_tree"));
+		this.open_channel(false, false, false, window.current_channel.get_root("main_tree"));
 	},
 	clipboard_page:function(){
-		this.open_channel(true, true, window.current_user.get_clipboard());
+		this.open_channel(true, true, false, window.current_user.get_clipboard());
 	},
-	open_channel: function(edit_mode_on, is_clipboard, root){
+	open_channel: function(edit_mode_on, is_clipboard, is_staging, root){
```

Comment:
It might be a good opportunity to do an 'Introduce Parameter Object' refactoring here, and combine the various boolean flags into a single object.